net-libs/gnome-online-accounts/gnome-online-accounts-3.46.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2 vala
+
+DESCRIPTION="GNOME framework for accessing online accounts"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
+
+LICENSE="LGPL-2+"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="debug gnome +introspection kerberos +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+# pango used in goaeditablelabel
+# libsoup used in goaoauthprovider
+# goa kerberos provider is incompatible with app-crypt/heimdal, see
+# https://bugzilla.gnome.org/show_bug.cgi?id=692250
+# json-glib-0.16 needed for bug #485092
+RDEPEND="
+	>=dev-libs/glib-2.67.4:2
+	sys-apps/dbus
+	introspection? ( >=dev-libs/gobject-introspection-0.6.2:= )
+	>=x11-libs/gtk+-3.19.12:3
+	>=net-libs/webkit-gtk-2.33.1:4 # TODO this needs webkitgtk-4.1, so gtk4 slot? # javascriptcoregtk-4.1 is this?
+	>=dev-libs/json-glib-0.16
+	>=app-crypt/libsecret-0.5
+	>=net-libs/libsoup-3.0:3
+	dev-libs/libxml2:2
+	net-libs/rest-0.9.0:1.0 # TODO needs PR #27383 to be merged first
+	kerberos? (
+		app-crypt/gcr:0=[gtk]
+		app-crypt/mit-krb5
+	)
+	x11-libs/pango # TODO why do we need this? I can not find it
```

Comment:
I also see no evidence of a pango dependency, in either 3.46.0 or 3.44.0. Let's remove it.